libs/ledgerjs/packages/react-native-hw-transport-ble/src/BleTransport.ts

```diff
@@ -527,31 +589,37 @@ export default class BluetoothTransport extends Transport {
 
     if (mtu > 23) {
       const mtuSize = mtu - 3;
-      log(
-        "ble-verbose",
-        `BleTransport(${String(this.id)}) mtu set to ${String(mtuSize)}`
-      );
+      log(TAG, `BleTransport(${this.id}) mtu set to ${mtuSize}`);
       this.mtuSize = mtuSize;
     }
 
     return this.mtuSize;
   }
 
+  /**
+   * Exposed method from the ble-plx library
+   * Request the connection priority for the given device.
+   * @param {"Balanced" | "High" | "LowPower"} connectionPriority: Connection priority.
+   * @returns {Promise<Device>} Connected device.
+   */
   async requestConnectionPriority(
     connectionPriority: "Balanced" | "High" | "LowPower"
```

Comment:
I think you can do `connectionPriority: keyof typeof ConnectionPriority` and remove the casting with `as` below (why `keyof typeof` and not `keyof` ? 🤷 TS enums 😢)

Reply:
What should I do? Sorry but I don't understand and have dyslexia